x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -54,10 +54,19 @@ export const AGENT_POLICY_SUMMARY_ROUTE = `${BASE_POLICY_ROUTE}/summaries`;
 export const ISOLATE_HOST_ROUTE = `${BASE_ENDPOINT_ROUTE}/isolate`;
 export const UNISOLATE_HOST_ROUTE = `${BASE_ENDPOINT_ROUTE}/unisolate`;
 
+const BASE_ENDPOINT_ACTION_ROUTE = `${BASE_ENDPOINT_ROUTE}/action`;
+
+/** Action Response Routes */
+export const ISOLATE_HOST_ROUTE_V2 = `${BASE_ENDPOINT_ACTION_ROUTE}/isolate`;
+export const RELEASE_HOST_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/release`;
+export const GET_RUNNING_PROCESS_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/get_running_processes`;
```

Comment:
this seems.. long

`/running_procs`?  `ps` might be too short and abbreviative.

Reply:
I'm good with `/running_procs`.